### PR TITLE
Feature/22 place graphic elements introduction final pages

### PR DESCRIPTION
**Pull request by baxita**
- fikk ikke til å sjekke reload-icon på dimmed-versjonen :/ derfor fulgte jeg plassering til arrow til introduction page bare plasserte den litt lengre ned fra top ...Mulig jeg må justere denne!!

### CODE REVIEW

**Comment by baxita** [comment]
NB!! ser ut til at endringene til feature 23/place graphic elements til Human health har blitt med her...(?)

File: index.html [hunk 1]
```diff
@@ -1,167 +1,161 @@
...
+      <button class="dimmed">Read more</button>
+      <img
+          id="reload"
           class="dimmed"
           src="assets/images/section_1/reload_arrow.svg"
-          alt=""
-        /> -->
-        <p class="dimmed direction direction-reload">Reload page</p>
-      </section>
+          alt="reload-icon"
```
Comment (baxita) [inline index.html:38]: vet ikke hvorfor alt teksten har havnet her ? flyttes til linje 39

File: assets/styles/index.scss [hunk 2]
```diff
@@ -68,13 +82,65 @@ section {
...
 // * Visibility
 .dimmed {
   display: none;
 }
-
+//! TO DO: ser ikke hvor den havner
+#reload {
+  position: absolute;
+  top: 88%;
+  left: 50%;
```
Comment (baxita) [inline assets/styles/index.scss:141]: skal redigeres til left: 48,9%

**Comment by baxita** [comment]
> Since you have commented problems that you think should be fixed, could you maybe just fix those and then update the PR? Or delete this one and create a new one when you have finished resolving the issue.

Sure, but since i also have commits here that belong to feature23/   is it best to delete the whole PR and do new commits in VS ?

**Comment by icanseetime** [comment]
> > Since you have commented problems that you think should be fixed, could you maybe just fix those and then update the PR? Or delete this one and create a new one when you have finished resolving the issue.
> 
> Sure, but since i also have commits here that belong to feature23/   is it best to delete the whole PR and do new commits in VS ?

The commits are connected to the branch, not the PR. The issues should be separated if you can do that, but you know your code the best and have to make that decision.

Also, you can see the dimmed/polluted elements by swapping the css inside the classes, like I showed you. Please do that and test the dimmed elements 👍

**Comment by baxita** [comment]
> > > Since you have commented problems that you think should be fixed, could you maybe just fix those and then update the PR? Or delete this one and create a new one when you have finished resolving the issue.
> > 
> > 
> > Sure, but since i also have commits here that belong to feature23/   is it best to delete the whole PR and do new commits in VS ?
> 
> The commits are connected to the branch, not the PR. The issues should be separated if you can do that, but you know your code the best and have to make that decision.
> 
> Also, you can see the dimmed/polluted elements by swapping the css inside the classes, like I showed you. Please do that and test the dimmed elements 👍

ok, so if I understand it right I can just close this PR  and revert the wrong belonging feature commits in VS code and do new commits with the "right" feature/branch again in VS-code? 

I tried comment-out  polluted and dimmed, but I had some watch sass bugs all day through ;)  that's why I couldn't always see  results of changes at times, but solved in the end :) I can fix the reload icon in the new commits!  BTW- the reload button; it should also be clickable?  in case , that also need to be fixed.

**Comment by baxita** [comment]
> > ok, so if I understand it right I can just close this PR  and revert the wrong belonging feature commits in VS code and do new commits with the "right" feature/branch again in VS-code?
> > I tried comment-out  polluted and dimmed, but I had some watch sass bugs all day through ;)  that's why I couldn't always see  results of changes at times, but solved in the end :) I can fix the reload icon in the new commits!  BTW- the reload button; it should also be clickable?  in case , that also need to be fixed.
> 
> Yep, that sounds great.
> 
> The functionality for both buttons will be fixed in later issues.

:) ok. Then I will close this PR !! and do some changes ! Thanks for advice 👍

**Pull request closed without merging**